src/frontend/src/optimizer/plan_node/logical_insert.rs

```diff
@@ -106,17 +107,41 @@ impl LogicalInsert {
     }
 
     pub(super) fn fmt_with_name(&self, f: &mut fmt::Formatter<'_>, name: &str) -> fmt::Result {
+        let verbose = self.ctx().is_explain_verbose();
         write!(
             f,
-            "{} {{ table: {}{} }}",
+            "{} {{ table: {}{}",
             name,
             self.table_name,
             if self.returning {
                 ", returning: true"
             } else {
                 ""
             }
-        )
+        )?;
+        if verbose {
+            write!(
+                f,
+                ", source to insert column mapping: [{}]",
```

Comment:
What about making this more concise, and following the `snake_case`? 🥵
```suggestion
                ", mapping: [{}]",
```